src/components/pagination/__test__/pagination.test.js

```diff
@@ -27,6 +27,14 @@ describe('Pagination component', () => {
     const component = renderer.create(<Pagination total={3} current={3} />);
     expect(component.toJSON()).toMatchSnapshot();
   });
+  it('All buttons should be disabled', () => {
+    const component = renderer.create(<Pagination total={3} current={1} disabled />);
+    expect(component.toJSON()).toMatchSnapshot();
+  });
+  it('previous button should be disabled when the prop prevDisabled is given', () => {
+    const component = renderer.create(<Pagination prevDisabled />);
```

Comment:
This test its not valid anymore right?

Reply:
Yeah it shouldn't be valid anymore; not sure why the tests passed. Will fix this asap.